SWIPs/swip-message_to_honey.md

```diff
@@ -48,19 +53,19 @@ This section describes the interaction between the nodes and the oracle in more
   }
 }
 ```
-* After `TTL` expires, nodes will query the price oracle for a new honeyPrices object. 
+* After `TTL` expires, nodes will query the price oracle for a new `honeyPrices` object. 
```

Comment:
Possible naming inconsistency.
Above we talk about the tuple `(price,validUntil)`, and here about TTL. I vote that we use always the same.

Reply:
TTL is up-to-date.